naga/fuzz/fuzz_targets/glsl_parser.rs

```diff
@@ -2,41 +2,32 @@
 
 #[cfg(enable_fuzzing)]
 mod fuzz {
+    use std::iter::FromIterator;
+
     use arbitrary::Arbitrary;
     use libfuzzer_sys::fuzz_target;
     use naga::{
         front::glsl::{Frontend, Options},
         FastHashMap, ShaderStage,
     };
 
-    #[derive(Debug, Arbitrary)]
-    enum ShaderStageProxy {
-        Vertex,
-        Fragment,
-        Compute,
-    }
-
-    impl From<ShaderStageProxy> for ShaderStage {
-        fn from(proxy: ShaderStageProxy) -> Self {
-            match proxy {
-                ShaderStageProxy::Vertex => ShaderStage::Vertex,
-                ShaderStageProxy::Fragment => ShaderStage::Fragment,
-                ShaderStageProxy::Compute => ShaderStage::Compute,
-            }
-        }
-    }
-
```

Comment:
I always love the idea of removing code not needed to maintain, ideal if we can find a way to do this in a separate PR (???)